ods_ci/tests/Resources/Page/DistributedWorkloads/WorkloadMetricsUI.resource

```diff
@@ -45,3 +57,293 @@ Check Expected String Equals
+    ELSE
+        ${memory_requested_gb_value}=    Evaluate    round( ${memory_requested_num} / (1024*1024*1024), 3)
+    END
+    RETURN  ${memory_requested_gb_value}
+
+Get Requested Resources Tooltip Text
+    [Documentation]    retuns tooltip text of requested resources chart
+    Wait Until Element Is Visible    xpath://*[contains(@style, 'fill: var(--pf-v5-chart-tooltip--Fill')]
+    ${tooltip_text} =  Get Text    xpath://*[contains(@style, 'fill: var(--pf-v5-chart-tooltip--Fill')]
+    RETURN    ${tooltip_text}
+
+Get Memory Requested Tooltip Text
+    [Documentation]    retuns tooltip text of Memory requested chart
+    Mouse Over    xpath=//*[@aria-label="Memory usage/requested"]
+    Wait Until Element Is Visible    xpath://*[starts-with(@id,'pf-tooltip-')]
+    ${memory_usage_hover_data} =  Get Text    xpath://*[starts-with(@id,'pf-tooltip-')]
+    @{memory_usage_list}    Split To Lines    ${memory_usage_hover_data}
+    ${memory_usage_line}    Set Variable    ${memory_usage_list}[0]
+    ${memory_requested_line}    Set Variable    ${memory_usage_list}[1]
+    ${memory_usage_value}     Set Variable    ${memory_usage_line}[16:21]
+    ${memory_requested_value}     Set Variable    ${memory_requested_line}[20:25]
+    RETURN    ${memory_usage_value}     ${memory_requested_value}
+
+Get CPU Requested Tooltip Text
+    [Documentation]    retuns tooltip text of CPU requested chart
+    Mouse Over    xpath=//*[@aria-label="CPU usage/requested"]
+    Wait Until Element Is Visible    xpath://*[starts-with(@id,'pf-tooltip-')]
+    ${cpu_usage_hover_data} =  Get Text    xpath://*[starts-with(@id,'pf-tooltip-')]
+    @{cpu_usage_list}    Split To Lines    ${cpu_usage_hover_data}
+    ${cpu_usage_line}    Set Variable    ${cpu_usage_list}[0]
+    ${cpu_requested_line}    Set Variable    ${cpu_usage_list}[1]
+    ${cpu_usage_value}     Set Variable    ${cpu_usage_line}[13:18]
+    ${cpu_requested_value}     Set Variable    ${cpu_requested_line}[17:18]
+    RETURN    ${cpu_usage_value}    ${cpu_requested_value}
+
+Create Ray Cluster Workload
+    [Documentation]    checks Distributed Worklaod Status Overview displaying correctly
+    ${result} =    Run Process    git clone ${CODEFLARE-SDK_REPO_URL} ${CODEFLARE-SDK_DIR}
+    ...    shell=true    stderr=STDOUT
+    Log To Console    ${result.stdout}
+    IF    ${result.rc} != 0
+        FAIL    Unable to clone codeflare-sdk repo
+    END
+
+    ${result} =    Run Process  virtualenv -p python3.9 ${VIRTUAL_ENV_NAME}
+    ...    shell=true    stderr=STDOUT
+    Log To Console    ${result.stdout}
+    IF    ${result.rc} != 0
+        FAIL    Unable to setup Python virtual environment
+    END
+
+    ${result} =    Run Process  source ${VIRTUAL_ENV_NAME}/bin/activate && cd ${CODEFLARE-SDK_DIR} && poetry env use 3.9 && poetry install --with test,docs && poetry run pytest -v -s ./tests/upgrade/raycluster_sdk_upgrade_test.py::TestMNISTRayClusterUp --timeout\=300 && deactivate
```

Comment:
If I get it right the SDK test is called here to provision Kueue resources and RayCluster CR.
It is not ideal as this test relies on external code, whose implementation can change.
From my perspective it would be better to provision Kueue resources using the kueue_workloads.sh script as done for other cases and create RayCluster CR for example from local YAML file.

Reply:
Sure will implement accordingly

Reply:
@Srihari1192 You can implement it as a separate PR and merge this PR, to keep changes clear and minimal.

Reply:
@sutaakar Sure, Thanks